### PR TITLE
Level polymorphism for Algebra.Construct.Zero and Algebra.Module.Construct.Zero

### PR DESCRIPTION
**Pull request by conal**
Addresses issue #1321.

### CODE REVIEW

**Comment by MatthewDaggitt** [comment]
Looks good, thanks! I'll leave it up for a few days and then merge it in.

File: src/Algebra/Module/Construct/Zero.agda [hunk 1]
```diff
@@ -10,61 +10,62 @@
...
-open import Data.Unit
-open import Level
+open import Data.Unit.Polymorphic
 
 private
   variable
     r s ℓr ℓs : Level
 
-leftSemimodule : {R : Semiring r ℓr} → LeftSemimodule R 0ℓ 0ℓ
+leftSemimodule : {R : Semiring r ℓr} → LeftSemimodule R c ℓ
```
Comment (gallais) [inline src/Algebra/Module/Construct/Zero.agda:25]: Is there a reason all of these have level indices `c ℓ`
whereas in `Algebra.Construct` all of the definition return
constructs using `ℓ ℓ`?

File: src/Algebra/Construct/Zero.agda [hunk 2]
```diff
@@ -13,53 +13,54 @@
...
 open import Algebra.Bundles
-open import Data.Unit
-open import Level using (Level; 0ℓ)
+open import Data.Unit.Polymorphic
 
 ------------------------------------------------------------------------
 -- Raw bundles
 
-rawMagma : RawMagma 0ℓ 0ℓ
+rawMagma : RawMagma ℓ ℓ
```
Comment (JacquesCarette) [inline src/Algebra/Construct/Zero.agda:26]: Is there something that forces these to be the same level?

File: src/Algebra/Module/Construct/Zero.agda (hunk 1 above)
Comment (conal) [reply src/Algebra/Module/Construct/Zero.agda:25]: Thanks for the eyes and the question! Now I'm asking myself the same question. I just changed `Algebra.Construct` to use `c ℓ` as well. When `make clean && make test` finish, I'll update this pull request.

File: src/Algebra/Construct/Zero.agda (hunk 2 above)
Comment (conal) [reply src/Algebra/Construct/Zero.agda:26]: I don't think so. Thanks for the review. See previous reply.

**Pull request merged**